authors/apps/articles/views.py

```diff
@@ -531,3 +537,62 @@ def delete(self, request, slug):
                 , status=status.HTTP_200_OK
             )
 
+
```

Comment:
Add comments and docstrings to this file as has been done in the rest.

Reply:
Noted